.github/scripts/sync_labels.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+import sys
+import requests
+import json
+from github import Github
+
+try:
+    github_token = os.environ["GITHUB_TOKEN"]
+except KeyError:
+    print("Please set the 'GITHUB_TOKEN' environment variable")
+    sys.exit(1)
+
+
+def parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--repository', type=str, required=True,
+                        help='Github repository name (e.g., scylladb/scylladb)')
+    parser.add_argument('--number', type=int, required=True,
+                        help='Pull request number to sync labels from linked issue')
+    parser.add_argument('--is_issue', action='store_true', help='Determined if label change is in Issue or not')
+    parser.add_argument('--label_action', type=str, choices=['opened', 'labeled', 'unlabeled'], required=True, help='Sync labels action')
+    return parser.parse_args()
+
+
+def get_pr_request_body(repo, number):
+    url = f"https://api.github.com/repos/{repo}/pulls/{number}"
+    headers = {
+        'Authorization': f'token {github_token}',
+        'Accept': 'application/vnd.github.v3+json'
+    }
+    response = requests.get(url, headers=headers)
+    if response.ok:
+        pr_details = response.json()
+        return pr_details['body']
+    else:
+        print("Failed to fetch PR details. Status code:", response.status_code)
+
+
+def get_linked_issues(pr_body):
+    pattern = re.compile(r'Fixes:? (?:#|https.*?/issues/)(\d+)', re.IGNORECASE)
```

Comment:
1. No `Closes`?
2. What about the `scylladb/scylladb#number` format? - see https://github.com/scylladb/jenkins/blob/master/backport-queue/commitparser.py#L4

Reply:
?

Reply:
> 1. No `Closes`?

`Closes` points to the PR and not to the Fixed issue. Also, we add this only after we close the PR, so it will not help us while it's open

> 2. What about the `scylladb/scylladb#number` format? - see https://github.com/scylladb/jenkins/blob/master/backport-queue/commitparser.py#L4

I will change it to match this pattern. It didn't work for me during my debugging so I had to modify it